RELEASE_NOTES.md

```diff
@@ -1,3 +1,45 @@
+## Cookstyle 6.17
+
+### 4 New Cops
+
+#### ChefDeprecations/ChefShellout
+
+The `ChefDeprecations/ChefShellout` cop detects the use of the `Chef::ShellOut` class which was removed in Chef Infra Client 13. It will autocorrect all occurences to instead use `Mixlib::ShellOut`, which is a drop-in replacement.
+
+`Enabled by default`: True
+
+`Autocorrects`: Yes
+
+#### ChefDeprecations/ExecutePathProperty
+
+The `ChefDeprecations/ExecutePathProperty` cop detects `execute` resources that use the `path` property, which was removed in Chef Infra Client 13. To set a path for use when executing you should instead set the `PATH` via the `environment` property.
```

Comment:
```suggestion
The `ChefDeprecations/ExecutePathProperty` cop detects `execute` resources that use the `path` property, which was removed in Chef Infra Client 13. To set a path for use when executing, you should instead set the `PATH` via the `environment` property.
```
Should the second mention of "path" be in ALL CAPS in the last sentence - "set the `PATH` via the `environment` property"?

Reply:
@tas50 ^ what about the question above?

Reply:
I think it's meant to be uppercase in the environment property, but let me check

Reply:
```
environment
    Ruby Type: Hash

    A Hash of environment variables in the form of ({'ENV_VARIABLE' => 'VALUE'}). Note: These variables must exist for a command to be run successfully.
````

Reply:
Gotcha. Thank you for clarifying! We're good on this.